instagram.user.styl

```
@@ -10,12 +10,17 @@
 @version      2.2.1
 @license      MIT
 
-@var text     ui_font    'Custom UI font' '"font_name"'
-@var text     emoji_font 'Custom emoji font' '"font_name"'
-@var checkbox ic_invert  'Enable icon inversion in dark mode' 1
-@var checkbox enable_cm  'Enable right-click context menu' 1
-@var checkbox enable_la  'Enable removal of logged-out annoyances' 1
-@var checkbox enable_sb  'Enable theme scrollbars' 1
+@var text     ui_font       'Custom UI font' '"font_name"'
+@var text     emoji_font    'Custom emoji font' '"font_name"'
+@var checkbox ic_invert     'Enable icon inversion in dark mode' 1
+@var checkbox enable_cm     'Enable right-click context menu' 1
+@var checkbox enable_la     'Enable removal of logged-out annoyances' 1
+@var checkbox custom_width  'Custom scrollbar width' 1
+@var select   sb_color      'Scrollbar colors' {
+        "Default": "default",
+        "Dark": "dark",
```

I think that `dark` fails because there is a [dark built-in function][0] in stylus-lang.

The `*` next to _theme default_ is to use this option as default.

[0]: https://stylus-lang.com/docs/bifs.html#darkcolor

```suggestion
        "None": "none",
        "Theme default*": "theme",
```